HighAvailabilityEncodingStreaming/README.md

```diff
@@ -99,7 +99,7 @@ Update the Key Vault configuration to grant access to user submitting test reque
 1. Open the Key Vault instance.
 1. Go to **access policies**.
 1. Click on **Add access policy**.
-1. Select the *Get* and *List* secret permissions.
+1. Select the *Get* and *List* and *Write* secret permissions.
```

Comment:
Yes, we should update this. I think this should be 
Select the *Get*, *List* and *Set* secret permissions.